pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -215,81 +301,101 @@ def create_model(df_sets, df_parameters, default={}):
 
     ## Define continuous variables ##
 
-    model.v_Z = Var(within=Reals, doc="Objective function variable [$]")
+    model.v_Z = Var(
+        within=Reals,
+        units=model.model_units["currency"],
+        doc="Objective function variable [currency]",
+    )
 
     model.v_F_Piped = Var(
         model.s_L,
         model.s_L,
         model.s_T,
         within=NonNegativeReals,
         initialize=0,
-        doc="Produced water quantity piped from location l to location l [bbl/week]",
+        units=model.model_units["volume"] / model.model_units["time"],
```

Comment:
Why do we convert everything to time and not decision period? If we use decision period we do not have to use the model.time_per_period in the constraints

Reply:
Good question. Both methods work. The way it is currently, all flow balances hold
flows in  [bbl/day] = flows out [bbl/day]. If we were to convert this to decision_period it would be :
7*flows in [bbl/day] = 7*flows out [bbl/day], which is the same. The only constraints where this does NOT hold are storage balance and costs. 

You bring up a good point that perhaps it is most intuitive for the decision period to always align with the model unit for time / perhaps there is only need for one of these.

Reply:
updated! model.model_units["time"] no longer exists. When referencing time for model units, model.decision_period is used instead. All constraints updated appropriately, the results were consistent.